config-builder/src/test/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/JmsDestinationEntityBuilderTest.java

```diff
@@ -325,7 +325,6 @@ private static void verifyInbound(Entity entity) {
         
         assertEquals("AUTOMATIC", jmsDestinationDetailProps.remove(INBOUND_ACKNOWLEDGEMENT_TYPE));
         assertEquals("AUTOMATIC", jmsDestinationDetailProps.remove(REPLY_TYPE));
-        assertEquals("true", contextPropertiesTemplateProps.remove(IS_HARDWIRED_SERVICE));
         assertEquals("my-service-id", contextPropertiesTemplateProps.remove(HARDWIRED_SERVICE_ID));
```

Comment:
Add  back this `assertEquals` statement.

If Inbound JMS destination is configured to be associated to a specific service, then `IS_HARDWIRED_SERVICE` must be set to `true`. Otherwise, it won't resolve to that specific service.